pkg/controller/build/buildrequest/buildrequest.go

```diff
@@ -227,6 +228,30 @@ func (br buildRequestImpl) renderContainerfile() (string, error) {
 	return out.String(), nil
 }
 
+// podToJob creates a Job with the spec of the given Pod
+func (br buildRequestImpl) podToJob(pod *corev1.Pod) *batchv1.Job {
+	// Set the backoffLimit to 3 so the job will retry 4 times before reporting a failure
```

Comment:
thought (non-blocking): I think we eventually might want to expose this value via the API.

Reply:
you mean via the MachineOSConfig? If so, it needs to be included in the API changes then right?

Reply:
Eventually, but it doesn't need to be done right now. Making additive changes to the API is much easier than making subtractive ones.